frontend/src/Frontend/Wallet.hs

```diff
@@ -81,10 +86,18 @@ import Frontend.KadenaAddress
 import Frontend.Storage
 import Frontend.Network
 
+import Frontend.UI.Widgets
+
+accountIsCreated :: Account key -> AccountCreated
+accountIsCreated = maybe AccountCreated_No (const AccountCreated_Yes) . _account_balance
```

Comment:
Any reason not to use `newtype AccountCreated = AccountCreated Bool` instead?
Then you could simply `accountIsCreated = AccountCreated . isJust . _account_balance`